app/localization/translated/be.json

```diff
@@ -119,6 +129,11 @@
   "MembersPage.permissionMap": "Карта правоў",
   "MembersPage.searchByName": "Шукаць па імя",
   "MembersPage.title": "Удзельнікі праекта",
+  "MembersPageToolbar.inviteUser": "Invite user",
+  "MembersPageToolbar.permissionMap": "Permission map",
+  "MembersPageToolbar.searchByName": "Search by name",
```

Comment:
Possible duplicate with `MembersPage`?
I think `MembersPage` has been recently renamed to `MembersPageToolbar`. Please check the code for the right localization key.

Reply:
fixed